watertap/core/membrane_channel_base.py

```diff
@@ -519,17 +551,32 @@ def eq_K(b, t, x, j):
                 == b.properties[t, x].diffus_phase_comp["Liq", j] * b.N_Sh_comp[t, x, j]
             )
 
-        @self.Constraint(
-            self.flowsheet().config.time,
-            self.length_domain,
-            self.config.property_package.solute_set,
-            doc="Sherwood number",
-        )
-        def eq_N_Sh_comp(b, t, x, j):
-            return (
-                b.N_Sh_comp[t, x, j]
-                == 0.46 * (b.N_Re[t, x] * b.N_Sc_comp[t, x, j]) ** 0.36
+        if sherwood_number_eq == SherwoodNumberEq.new:
+
+            @self.Constraint(
+                self.flowsheet().config.time,
+                self.length_domain,
+                self.config.property_package.solute_set,
+                doc="Sherwood number",
             )
+            def eq_N_Sh_comp(b, t, x, j):
+                return b.N_Sh_comp[t, x, j] == 2.401 * (
+                    (b.N_Re[t, x] * b.N_Sc_comp[t, x, j]) ** 0.297
+                ) * ((b.channel_length / b.dh) ** -0.279)
```

Comment:
Reading the original source, it seems like _x_ is the current position *in* the channel. So `length` should be multiplied by `x` here, I think. Probably also need to consider the backwards case carefully.

@adam-a-a thoughts?